### ISSUE TITLE
Failed to update Cache Error

### ISSUE THREAD

**Issue by aparker21** [owntracks/quicksetup#73]
I first ran bootstrap.sh and everything was running fine.  I was seeing green lines and everything was running smooth.  Somehow the first line in the Configuration.yaml file was no longer commented out. I fixed that and ran the ./bootsrap.sh command again.  It now wont' get past the install of the OwnTracks repository and says in Pink Letters: [WARNING]: Failed to update cache after 1 due to E: Failed to fetch http://deb.debian.org/debian/dists/bookworm/InRelease  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?), E: The repository 'http://deb.debian.or/debian bookworm InRelease' is no longer signed.  . . .. . . . It continues on for a few more lines and then repeats this for 2, 3, 4, and 5 seconds.  

Then it gives me a Red Error, "fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to update apt cache after 5 retries: E: Failed to fetch http://deb.debian.org/debian/dists/bookworm-updates/InRelease   Clearsigned file isn't valid. . . . .etc.  

I know this says the debian distro in the errors, but why did it work the first time I ran bootstrap.sh?  Why is it only now getting the error after the bootstrap failed due to the first line not being commented in the configuration.yaml file??  Any thoughts on how to fix this so I can get Owntracks up and running?

I'm running Debian 12 (bookworm) on a Raspberry pi 4
I have updated it multiple times and basically reinstalled it twice and still get the error.  Any help is very much appreicated!!  I am incredibly novice to Linux so I appreciate your help!

**Comment by jpmens** [comment]
> Failed to update apt cache after 5 

This is some issue with the repository and unfortunately out of our control. Ensure you have connectivity. Maybe try a manual `sudo apt update` to see if your system can access the Debian repo.

**Comment by aparker21** [comment]
I tried everything I could think of and could not get it to work.  I finally looked at my pfsense router and found that debian.org was being blocked. . . why it wasn't blocked the very first time i ran the sudo ./bootstrap.sh is beyond me, but that seems to have solved the problem above. . . but now I am getting an error in regards to Let's encrypt.  It appears to be due to my domain name not ending with a valid public suffix (TLD).  I will close this issue and look around for help on forums for that.  Thank you for the reply @jpmens
